app/src/main/res/layout/layout_profile_header.xml

```diff
@@ -10,6 +11,22 @@
   app:layout_constraintTop_toTopOf="parent"
   app:layout_constraintVertical_bias="0.0">
 
+  <Button
+      android:id="@+id/signout_button"
+      android:layout_width="90dp"
+      android:layout_height="50dp"

```

Comment:
It would be better to use `"wrap_content"` instead of hardcoding the button size.